.vscode/launch.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
a co to za pliki konfiguracyjne z vscode?

dorzuć do gitignore
dziwne że ten domyślny tego nie wykrywa...

Reply:
te konfiguracje są przydatne, jakby ktoś chciał to uruchomić za pomocą omnisharpa, specjalnie zacommitowalem, bo nic nie psuje, a sam to konfigurowalem zeby dobrze się uruchamiało